charts/astronomer/templates/commander/commander-role.yaml

```diff
@@ -43,7 +43,7 @@ rules:
   verbs: ["*"]
 - apiGroups: [""]
   resources: ["persistentvolumeclaims"]
-  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch"]
+  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch", "patch"]
```

Comment:
Do you think we should make a test for this and the persistentvolumes rule, specifically that these exact permissions are being assigned? The important thing being to make sure no additional permissions are assigned.

Reply:
IMHO it doesn't seem like that important of a test. If this file were dynamic and not mostly just hard coding I'd feel otherwise.